docs/data-model.asciidoc

```diff
@@ -378,6 +427,40 @@ include::./data-streams.asciidoc[tag=metrics-data-streams]
 
 See <<apm-data-streams>> to learn more.
 
+[float]
+==== Aggregated metrics: limits and overflows
+
+For all aggregated metrics, namely transaction, service-transaction, service-destination, and service-summary metrics,
+there are limits on the number of unique groups tracked at any given time.
+
+[float]
+===== Limits
+
+For transaction metrics, the limits are 1000 services per GB of APM Server, 5000 transaction
+groups per GB of APM Server, and 500 transaction groups per service per GB of APM Server.
```

Comment:
Maybe clarify a bit further the difference between the global limit of 5000 transaction groups and the per-service limit of 500 transaction groups.

```suggestion
For transaction metrics, the limits are 1000 services per GB of APM Server, and 5000 transaction
groups per GB of APM Server. Additionally, each service may only consume up to 10% of the transaction groups, which is 500 transaction groups per service per GB of APM Server.
```